global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.100",
+    "version": "8.0.100-preview.7.23376.3",
```

Comment:
Use RC.2?

Reply:
That would require installing that on build agents, and I didn't think that was necessary,